nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_and_recovery.xml

```diff
@@ -12,15 +12,13 @@
           <RecoveryNode number_of_retries="1" name="ComputePathToPose">
             <ComputePathToPose goal="{goal}" path="{path}" planner_id="GridBased"/>
             <ReactiveFallback name="ComputePathToPoseRecoveryFallback">
-              <GoalUpdated/>
```

Comment:
Can you also remove this from the navigate through poses BT?

Reply:
Done